internal/security-governance-system/webapp/src/main/resources/sgs-webapp-config.yaml

```diff
@@ -0,0 +1,38 @@
+#Security Governance System webapp configuration.
+
+clientId:
+clientSecret:
```

Comment:
This must be an encrypted value. Program must decrypt and use.

Reply:
This will be implemented in the next phase.

Reply:
This is tracked with https://github.com/wso2/security-tools/issues/148